app/components/img-selector.js

```diff
@@ -15,6 +16,9 @@ export default Ember.Component.extend({
                 }
             };
             reader.readAsDataURL(e.target.files[0]);
+        },
+        clickInput: function() {
+            this.$().find('.img-selector-input').click();
```

Comment:
Not sure what you mean by this- you can use class selectors with JQuery. It's even in their tutorial:
https://learn.jquery.com/using-jquery-core/faq/how-do-i-select-an-item-using-class-or-id/

And firing a click event for every item on the page (whenever you click one element) seems less desirable than restrictive design. Presumably if you wanted multiple image selectors, you'd use multiple isolated components.